docs/sources/panels-visualizations/query-transform-data/transform-data/index.md

```diff
@@ -1007,6 +1007,25 @@ This transformation is available in Grafana 10.1+ as an alpha feature.
 
 Use this transformation to format the output of a time field. Output can be formatted using (Moment.js format strings)[https://momentjs.com/docs/#/displaying/]. For instance, if you would like to display only the year of a time field the format string `YYYY` can be used to show the calendar year (e.g. 1999, 2012, etc.).
 
+### Format string
+
+{{% admonition type="note" %}}
+This transformation is available in Grafana 10.2+ and can be enabled using the `formatString` feature toggle.
+{{% /admonition %}}
+
+Use this transformation to format the output of a string field. You can format output in the following ways:
+
+- Upper case - Formats the entire string in upper case characters.
+- Lower case - Formats the entire string in lower case characters.
+- Sentence case - Formats the the first character of the string in upper case.
+- Title case - Formats the first character of each word in the string in upper case.
+- Pascal case - Formats the first character of each word in the string in upper case and doesn't include spaces between words.
+- Camel case - Formats the first character of each word in the string in upper case, except the first word, and doesn't includes spaces between words.
```

Comment:
Small typo fix (probably that I introduced!)
```suggestion
- Camel case - Formats the first character of each word in the string in upper case, except the first word, and doesn't include spaces between words.
```

Reply:
I should've double checked. Thank you for catching it!